### ISSUE TITLE
getting new relic to work on windows?

### ISSUE THREAD

**Issue by hanleybrand** [blackboard/newrelic-blackboard-learn#1]
I saw your talk at bbworld and I'm trying to get the new relic agent installed on our test system - I think I misunderstood what was said at the presentation (or since we use windows, maybe something is different there)

After unzipping the agent.zip from new relic I put the dir in `E:\blackboard\apps\tomcat` which is where `E:\blackboard\apps\tomcat\bin\catalina.bat` indicates it will look for it on startup - is this incorrect?  -- 

```
set JAVA_OPTS=%JAVA_OPTS% -javaagent:"E:/blackboard/apps/tomcat/newrelic/newrelic.jar"
```

Basically the server restarts fine but there's no indication that new relic.jar is getting loaded; the only thing in the logs is a "tomcat.util.scan.DefaultJarScanner.jarsToSkip =" that includes the newrelic-api.jar

And when that is working, does MessageQueueHandler.xml go directly into the \newrelic dir or does it need to go into \newrelic\extensions ?

Thanks for any insight you can give - is it possible something additional needs to be done for windows servers?

**Comment by ntatsumi** [comment]
Hi Peter,

I've updated the README with instructions on how to install New Relic Java application agent on Blackboard Learn April 2014 release and above:
https://github.com/blackboard/newrelic-blackboard-learn/blob/master/README.md

FYI - In the newer versions of the New Relic agent, I found a compatibility issue with the Java security manager, which Blackboard Learn uses. I'm working with New Relic to address this.

Best,
Nori
